skyportal/handlers/api/source.py

```diff
@@ -354,6 +354,7 @@ def get_sources(
     has_spectrum=False,
     has_followup_request=False,
     sourceID=None,
+    nonsourceIDs=None,
```

Comment:
Maybe `rejectedSourceIDs` or something like that?

Reply:
This is a much nicer name indeed.